app/models/framework/management_charge_calculator/flat_rate.rb

```diff
@@ -34,7 +36,7 @@ def overridding_calculation_column?
 
       def cast_to_decimal(column_value)
         case column_value
-        when String then BigDecimal(column_value)
+        when String then BigDecimal(column_value.gsub(NUMBER_WHITELIST, ''))
         when Numeric then column_value.to_d
```

Comment:
It's still making me twitch that we're doing the conversion in two different ways, when it seems like `string.to_d` also works for cleaned up strings.

(I tried replacing `column_value.to_d` with `BigDecimal(column_value)`, and it requires precision if it's already a numeric value. /o\)

Reply:
Good point! Updated so both call `to_d`.